android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -39,7 +39,8 @@
 public class WebRTCModule extends ReactContextBaseJavaModule {
     static final String TAG = WebRTCModule.class.getCanonicalName();
 
-    PeerConnectionFactory mFactory;
+    private final ReactApplicationContext reactContext;
```

Comment:
The context is also cached similarly in `JavaAudioDeviceModule` class, though that's using `android.content.Context` rather than `com.facebook.react.bridge.ReactApplicationContext`. 

We could convert this to exactly the same context type as `JavaAudioDeviceModule`?